src/functions/formatSort.js

```diff
@@ -5,22 +5,29 @@ var find = require('./find');
 /**
  * Transform sort format from user friendly notation to lodash format
  * @param {string[]} sortBy array of predicate of the form "attribute:order"
+ * @param {string[]} [defaults] array of predicate of the form "attribute:order"
  * @return {array.<string[]>} array containing 2 elements : attributes, orders
  */
 module.exports = function formatSort(sortBy, defaults) {
+  defaults = (defaults || []).map(function(sort) {
```

Comment:
Not sure the default value is useful, IIRC it's always called with a value.

Reply:
Hierarchical calls it without defaults I think